client/my-sites/people/team-list/team.jsx

```diff
@@ -125,18 +125,24 @@ class Team extends React.Component {
 	_fetchNextPage = () => {
 		const offset = this.props.users.length;
 		const fetchOptions = Object.assign( {}, this.props.fetchOptions, { offset: offset } );
-		analytics.ga.recordEvent( 'People', 'Fetched more users with infinite list', 'offset', offset );
+		this.props.recordGoogleEvent(
+			'People',
+			'Fetched more users with infinite list',
+			'offset',
+			offset
+		);
 		debug( 'fetching next batch of users' );
 		fetchUsers( fetchOptions );
 	};
 
-	_getPersonRef = user => {
-		return 'user-' + user.ID;
-	};
+	_getPersonRef = user => 'user-' + user.ID;
```

Comment:
It'd be good to chop off the `_`s from the variable names here as per the [guidlines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#react-components). Not a blocker though! I wouldn't want to see that get in the way of merging :)